Makefile

```diff
@@ -241,7 +242,13 @@ buildEcsDeployer:
 
 ## Deploy all infrastructure
 deploy: dockerCheck dockerLogin cleanMisc modelCheck buildEcsDeployer
-ifdef PROFILE
+ifneq (,$(findstring true, $(HEADLESS)))
+	if [ -z ${PROFILE} ]; then \
```

Comment:
Is this indentation actually off or just how github is showing it?

Reply:
makefile syntax can be quirky
- if you want a line of code to be interpreted as make code it must only be indented with spaces.
- if you want a line of code to be interpreted as bash code it must only be indented with tabs